### ISSUE TITLE
Mixed Underscores and dashes in install binary

### ISSUE THREAD

**Issue by abuehrle** [weaveworks/flux#342]
Should have a convention on which one to use and stick to it.  :) confusing and hard to notice...

```
https://github.com/weaveworks/flux/releases/download/master-6cc08e4/fluxctl-linux-amd64
```

```
https://github.com/weaveworks/flux/releases/download/master-0d109dd/fluxctl_linux_amd64
```

**Comment by paulbellamy** [comment]
Will be fixed as part of: https://github.com/weaveworks/flux/issues/342

**Comment by philwinder** [comment]
Fixed in #248
